alembic.rst

```diff
@@ -0,0 +1,84 @@
+Alembic
+=======
+Alembic is a database migration tool for usage with SQLAlchemy Database. The full documentation can be found here: 
+In invenio the alembic environment is already defined in invenio-db
+
+Creating new revisions
+----------------------
+
+When working with an invenio module that already has alembic revisions, after modifying a model we can create a new alembic revision with:
+
+.. code-block:: console 
+
+    $ invenio alembic revision "Added some column." --path alembic_directory_path
+
+A short message describing the changes is required and the path parameter should point to the alembic directory of the module. If the path is not given the new revision will be placed in the invenio_db/alembic directory.
+
+The directories accepted are registered by the invenio_db.alembic entrypoints.
+If the DB has not been updated to show the last changes, there is some alembic revision that needs to be applied by:
+
+.. code-block:: console 
+
+    $ invenio alembic upgrade heads
+
+To create alembic revisions for a module that didn't have any before, we first need to create a branch for it.
+
+Show current state
+------------------
+
+log
+
+history?
+
+current
+
+
+Adding alembic support to existing modules
+------------------------------------------
+The following procedure will add alembic support to invenio-unicorn for demonstration purposes.
+
+We create the alembic directory with:
+
+.. code-block:: console
+
+   $ invenio alembic mkdir
+
+in invenio_unicorn and add the invenio_db.alembic entrypoint in setup.py:
```

Comment:
You can remove `in invenio_unicorn and`. The sentence is hard to read if it is split by code.
=> Add the invenio_db.alembic entrypoint in setup.py: